### DISCUSSION TITLE
Where can someone with an IQ < 70 learn NASM

### DISCUSSION THREAD

**Discussion by ghost** [community/community#66765] [Programming Help]
### Body

I want to learn assembly but I don’t know if I really can because of what I have. I think I gotten better at it 
Have I learned anything?

```ASM
section .data
  num1: dd 5
  num2: dd 5
  result: dd 0
section .text
  global _start
_start:
 mov eax, dword[num1]
 mov edx, dword[num2]
 add edx, eax
 mov dword[result], eax
```

I wanted to know if someone like me can learn NASM

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by theintrigued** [comment]
Learning NASM (Netwide Assembler) or any programming language isn't solely determined by your IQ. Intelligence is just one factor that can affect the learning process. Many other factors, such as dedication, persistence, the quality of learning resources, and the methods you use, play a significant role in your ability to learn.

- **Start with the Basics**: You've already written a simple assembly program, which is a good start. Continue by studying the fundamental concepts of assembly language, such as registers, instructions, memory addressing modes, and data types.

- **Practice Regularly**: Assembly language programming requires practice. Try writing more complex programs and experiment with different instructions and concepts.

- **Use Online Resources**: There are many online tutorials, courses, and forums where you can find information and ask questions about assembly programming. Websites like Stack Overflow, tutorials on YouTube, and dedicated assembly language forums can be valuable resources. (here is one that I tried in my early days, https://www.youtube.com/watch?v=DNPjBvZxE3E)

- **Learn from Books**: Consider getting a book on assembly programming. There are many excellent books available that cover various assembly languages and provide detailed explanations and examples.

- **Join a Community**: Participate in online communities or forums where people discuss assembly programming. Asking questions and learning from experienced programmers can be very helpful.

- **Be Patient and Persistent**: Learning assembly language can be challenging, but don't get discouraged. It takes time and practice to become proficient. Stay patient and keep working on it.

Hope this helps!!
